src/components/search/PageSearchModal/index.js

```diff
@@ -113,13 +113,12 @@ module.exports = React.createClass({
     }
   },
 
-  selectHandler: function(page) {
+  selectHandler: function(page, event) {
```

Comment:
@slwen any thoughts here? This was my "it'll work" suggestion... got any better ideas?